file-manager-service/Protos/fileManager.proto

```diff
@@ -7,7 +7,7 @@ option csharp_namespace = "Gov.Lclb.Cllb.Services.FileManager";
 package filemanager;
 
 // The files service definition.
-service FileManager { 
+service FileManager {
```

Comment:
We need to ensure that this change did not affect any of the other services that use the file manager - please check with me and we'll go over that.